packages/wallets/src/components/Dropzone/__tests__/Dropzone.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Dropzone from '../Dropzone';
+
+describe('Dropzone', () => {
+    beforeAll(() => {
+        global.URL.createObjectURL = jest.fn();
```

Comment:
if global variable are updated, then they need to be cleaned up afterwards, otherwise you gonna get state leak